core/pom.xml

```diff
@@ -135,6 +138,18 @@
             <artifactId>evomaster-dbconstraint</artifactId>
         </dependency>
 
+        <!--for token parser-->
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>${nlp.version}</version>
```

Comment:
versions should be in the root pom, not in the submodules. see what I wrote recently in `for_developers.md`

Reply:
thanks, fixed